aws-kendra-index/src/main/java/software/amazon/kendra/index/Translator.java

```diff
@@ -123,6 +131,8 @@ static ResourceModel translateFromReadResponse(final DescribeIndexResponse descr
                 .queryCapacityUnits(describeIndexResponse.capacityUnits().queryCapacityUnits())
                 .build());
       }
+    } if (describeIndexResponse.hasUserTokenConfigurations()) {
```

Comment:
Do you need this check? Looks like `translateFromSdkUserTokenConfigurations` does the check for you - I'd just have it in one place. Another reason I'd remove the check is having it results in code coverage going down unless we have tests that check both branches.

Reply:
Updated in : https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-kendra/pull/81

GitHubGitHub